docs/site/Fields-filter.md

```diff
@@ -0,0 +1,87 @@
+---
+title: 'Fields filter'
+lang: en
+keywords: LoopBack 4.0, LoopBack 4, fields
+layout: readme
+source: loopback-next
+file: packages/metadata/README.md
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Fields-filter.html
+---
+
+A _fields_ filter specifies properties (fields) to include or exclude from the results.
+
+### REST API
+
+
+**The <true|false> does not work for url as the boolean is being concerted to string, which is always trru**
+<pre>
+filter[fields][<i>propertyName</i>]=<true>&filter[fields][<i>propertyName</i>]=<true>...
+</pre>
+
+Note that to include more than one field in REST, use multiple filters.
+
+You can also use [stringified JSON format](Querying-data.html#using-stringified-json-in-rest-queries) in a REST query.
+
+### Node API
```

Comment:
`Node` -> `Node.js`

Reply:
LB3 uses `Node`, WDYT?

Reply:
you're right. just realized they're `Node`. hm. i thought Node.js would be the more proper name. But good to leave it as-is.

Reply:
I changed all to Node.js